crates/core/src/database_logger.rs

```diff
@@ -125,7 +125,7 @@ impl DatabaseLogger {
 
     pub fn filepath(address: &Address, instance_id: u64) -> PathBuf {
         let root = crate::stdb_path("worker_node/database_instances");
-        root.join(address.to_hex())
+        root.join(&*address.to_hex())
```

Comment:
What's the significance of these reborrows (here and in database_instance_context.rs)?

Reply:
Oh, yes. There was another part of this that I ended up splitting off that changed stuff around Address and Identity hex stuff.